docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/SsaDetectSpike.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class SsaDetectSpike
```

Comment:
>SsaDetectSpike [](start = 24, length = 14)

ditto: rename to DetectSpikeBySsa. Make sure to update the filename inside xml CDATA.